che--typescript/src/internal/action/impl/list-workspaces-action.ts

```diff
@@ -64,7 +66,18 @@ export class ListWorkspacesAction {
                     // Create Ascii array
                     let rows : Array<Array<string>> = new Array<Array<string>>();
                     workspaceDtos.forEach((workspaceDto : any) => {
-                        rows.push([workspaceDto.getConfig().getName(), workspaceDto.getId(), workspaceDto.getStatus()]);
+                        let syncAgent : string = "";
+                        if (this.syncAgentCheck === true){
+                            let defaultEnv:string = workspaceDto.getConfig().getDefaultEnv();
+                            let agents:Array<string> = workspaceDto.getConfig().getEnvironments().get(defaultEnv).getMachines().get("dev-machine").getAgents();
+                            if (agents.indexOf('org.eclipse.che.unison') !== -1) {
+                                syncAgent = "(Sync Agent)";
+                            }
+                            else{
+                                syncAgent = "(No Sync Agent)";
+                            }
+                        }
+                        rows.push([workspaceDto.getConfig().getName()+syncAgent, workspaceDto.getId(), workspaceDto.getStatus()]);
```

Comment:
Yes I added a parameter `--sync-agent` to `list-workspaces-action` container from che-mount entrypoint.sh file. This means that the sync agent notification will only occur when using che-mount.